desenvolvimento-back-end/bloco_21/dia_01/exercises.sql

```diff
@@ -0,0 +1,54 @@
+# Exercício 1
+SELECT MAX(SALARY) AS 'MAIOR SALÁRIO' FROM hr.employees;
+
+# Exercício 2
+SELECT MAX(SALARY) - MIN(SALARY) FROM hr.employees;
+
+# Exercício 3
+SELECT JOB_ID, AVG(SALARY) AS 'avg_salary' FROM hr.employees GROUP BY JOB_ID ORDER BY `avg_salary` DESC;
+
+# Exercício 4
+SELECT SUM(SALARY) FROM hr.employees;
+
+# Exercício 5
+SELECT MAX(SALARY) AS 'Maior Salário', MIN(SALARY) AS 'Menor Salário', SUM(SALARY) AS 'Total Salários', ROUND(AVG(SALARY), 2) AS 'Média Salarial' FROM hr.employees;
+
+# Exercício 6
+SELECT JOB_ID, COUNT(*) FROM hr.employees WHERE JOB_ID = 'IT_PROG';
+
+# Exercício 7
+SELECT JOB_ID, SUM(SALARY) FROM hr.employees GROUP BY JOB_ID;
+
+# Exercício 8
+SELECT JOB_ID, SUM(SALARY) FROM hr.employees GROUP BY JOB_ID HAVING JOB_ID = 'IT_PROG';
+
+# Exercício 9
+SELECT JOB_ID, AVG(SALARY) AS 'avg_salary' FROM hr.employees GROUP BY JOB_ID HAVING JOB_ID NOT IN ('IT_PROG') ORDER BY `avg_salary` DESC;
+
+# Exercício 10
+SELECT DEPARTMENT_ID, AVG(SALARY), COUNT(*) FROM hr.employees GROUP BY DEPARTMENT_ID HAVING COUNT(*) > 10;
+
+# Exercício 11
+SET SQL_SAFE_UPDATES = 0;
+UPDATE hr.employees SET PHONE_NUMBER = REPLACE(PHONE_NUMBER, '515', '777') WHERE PHONE_NUMBER LIKE '515%';
```

Comment:
Um ponto de atenção para essa query é que se houver algum número que tenha 515 em alguma outra parte além do início, essa parte também será alterada. Por exemplo, '515-888-515' vira '777-888-777'.

Reply:
> Um ponto de atenção para essa query é que se houver algum número que tenha 515 em alguma outra parte além do início, essa parte também será alterada. Por exemplo, '515-888-515' vira '777-888-777'.

@paulohbsimoes, obrigada pelo alerta! Preciso utilizar o substring né?